backend/sqlite/sqlite.go

```diff
@@ -403,6 +403,9 @@ func (be *sqliteBackend) CreateOrchestrationInstance(ctx context.Context, e *bac
 
 	var instanceID string
 	if err := be.createOrchestrationInstanceInternal(ctx, e, tx, &instanceID); err != nil {
+		if err == backend.ErrSkipExistInstance {
+			return nil
```

Comment:
We should log a warning message here. Something like: "An instance with ID '%s' already exists; dropping duplicate create request"